src/target.rs

```diff
@@ -365,6 +370,7 @@ impl Target {
             Arch::Aarch64 => "aarch64",
             Arch::Armv6L => "armv6l",
             Arch::Armv7L => "armv7l",
+            Arch::Powerpc => "powerpc",
```

Comment:
I'm not totally sure what does `sysconfig.get_platform()` on powerpc Linux, @ionenwks any ideas?

Reply:
I've been told "linux-ppc"